addons/actions/src/preview.js

```diff
@@ -24,8 +24,10 @@ export function action(name) {
     });
   };
 
+  // IE11 may return an undefined descriptor
+  const nameDescriptor = Object.getOwnPropertyDescriptor(handler, 'name');
   // This condition is true in modern browsers that implement Function#name properly
-  const canConfigureName = Object.getOwnPropertyDescriptor(handler, 'name').configurable;
+  const canConfigureName = nameDescriptor && nameDescriptor.configurable;
```

Comment:
We can also try `!nameDescriptor || nameDescriptor.configurable`. As far as I remember, IE11 had no problem with overriding function name

Reply:
Good call, I tested this suggestion in IE11 and it worked. Pushed a new commit, please have a look.